projects/ks89/angular-modal-gallery/src/lib/components/previews/previews.component.spec.ts

```diff
@@ -956,6 +956,41 @@ describe('PreviewsComponent', () => {
       
     });
 
+    it('should always display previews navigation arrows, in infinite sliding mode (nbPreviews < nbImages)', () => {
```

Comment:
Addresses: 4. in infinite sliding, the left and right preview arrows should be always visible (like the left and right image arrows), except if nbPreviews < images.length